qa/suites/rados/dashboard/tasks/dashboard.yaml

```diff
@@ -39,6 +39,7 @@ tasks:
         - tasks.mgr.dashboard.test_logs
         - tasks.mgr.dashboard.test_mgr_module
         - tasks.mgr.dashboard.test_monitor
+        - tasks.mgr.dashboard.test_orchestrator
```

Comment:
What about ``test_rbd_mirroring`` which is mentioned in the tracker issue?

Reply:
Please have a look at: https://tracker.ceph.com/issues/42244#note-2
I don't have an approach for _test_rbd_mirroring_ at the moment. I decided to create the PR for the orchestrator suite already so that we can at least run these tests. Besides that I'm open for input ;)

Reply:
Thx for clarification.